addons/website_sale/views/templates.xml

```diff
@@ -1854,27 +1873,7 @@
         </t>
     </template>
 
-    <template id="address_b2b" inherit_id="address" name="Show b2b fields">
-        <xpath expr="//div[@id='div_phone']" position="after">
-            <div class="w-100"/>
-            <t t-set='vat_warning' t-value="'vat' in checkout and checkout['vat'] and not can_edit_vat" />
-            <t t-if="mode == ('new', 'billing') or (mode == ('edit', 'billing') and (can_edit_vat or 'vat' in checkout and checkout['vat']))">
-                <div t-attf-class="mb-3 #{error.get('company_name') and 'o_has_error' or ''} col-lg-6 mb-0">
-                    <label class="col-form-label fw-normal label-optional" for="company_name">Company Name</label>
-                    <input type="text" name="company_name" t-attf-class="form-control #{error.get('company_name') and 'is-invalid' or ''}" t-att-value="'commercial_company_name' in checkout and checkout['commercial_company_name'] or 'company_name' in checkout and checkout['company_name']" t-att-readonly="'1' if vat_warning else None" />
-                    <small t-if="vat_warning" class="form-text text-muted d-block d-lg-none">Changing company name is not allowed once document(s) have been issued for your account. Please contact us directly for this operation.</small>
-                </div>
-                <div t-attf-class="mb-3 #{error.get('vat') and 'o_has_error' or ''} col-lg-6 div_vat mb-0">
-                    <label class="col-form-label fw-normal label-optional" for="vat">TIN / VAT </label>
-                    <input type="text" name="vat" t-attf-class="form-control #{error.get('vat') and 'is-invalid' or ''}" t-att-value="'vat' in checkout and checkout['vat']" t-att-readonly="'1' if vat_warning else None"/>
-                    <small t-if="vat_warning" class="form-text text-muted d-block d-lg-none">Changing VAT number is not allowed once document(s) have been issued for your account. Please contact us directly for this operation.</small>
-                </div>
-                <div t-if="vat_warning" class="col-12 d-none d-lg-block mb-1">
-                    <small class="form-text text-muted">Changing company name or VAT number is not allowed once document(s) have been issued for your account. Please contact us directly for this operation.</small>
-                </div>
-            </t>
-        </xpath>
-    </template>
+    <template id="address_b2b" inherit_id="address" name="Show b2b fields" />
```

Comment:
We can reduce the diff if we add a new template; the previous `address_b2b` template becomes only an option and a new template `address_b2b_fields` (or maybe content? doesn't really matter to me that much) and t-call that template within the condition added in `website_sale.address` instead.

```suggestion
    <template id="address_b2b" inherit_id="address" name="Show b2b fields"/>
    
    <template id="website_sale.address_b2b_fields"/>
```